src/axom/core/Array.hpp

```diff
@@ -1001,6 +1001,15 @@ inline void Array<T, DIM, SPACE>::updateNumElements(IndexType new_num_elements)
 {
   assert(new_num_elements >= 0);
   assert(new_num_elements <= m_capacity);
+
+  int new_elems = new_num_elements - m_num_elements;
+  for(int ielem = 0; ielem < new_elems; ielem++)
+  {
+    // TODO: what to do here when T isn't default-constructible?
```

Comment:
Do we capture these TODO items sprinkled in the code as GitHub issues?

Reply:
This particular TODO is the subject of https://github.com/LLNL/axom/pull/709